### DISCUSSION TITLE
First time using Fan Control, experiencing a potential issue

### DISCUSSION THREAD

**Discussion by Art3mis-Ghub** [Rem0o/FanControl.Releases#1991] [Q&A]
Hi there,

This is my first time using Fan Control so bear with me if my question is a little dumb :)

When Fan Control Setup checked for fans, it found five in total; 

- the AiO ''fan'' (two connected to the same header, but obviously only counts as one)
- An unknown ''Fan 2'' (did not get a response from the controller)
- another unknown ''Fan 5'' (did get a response from the controller, but it shows up as 0 RPM regardless of speed %)
- Two GPU fans (Guess it registers each fan separately for GPUs?)

Now, as for my dumb question: Why would it show an unknown ''Fan 2'' that has no controller and no RPM, and why does ''Fan 5'' show up, it has a controller but no RPM, changing it does nothing?

I have a theory which I'll share with you, but again, bear in mind that I don't know how these things works:
Could it be that ''Fan 2'' is the Built-in controller that my Case has? (a thingamajig where I can connect multiple fan and RGB cables to that then extends to the mobo)
and lastly, Could ''Fan 5'' be the AiO pump that is connected directly to one of the Sys_Fan headers? Seeing as it does have controls, but no RPM it would make sense, at least to me.

Thanks in advance for any help!

**Comment by Rem0o** [comment]
Motherboards use off the shelf control chips to control/monitor fan headers.  It is these chips are that interfaced in the software. Your motherboard might not use all the channels available on these chips, even though you can still address them from the software.
